rasa/utils/tensorflow/tf_models.py

```diff
@@ -291,8 +280,7 @@ def _should_evaluate(
 
     @staticmethod
     def batch_to_model_data_format(
-        batch: Union[Tuple[np.ndarray], Tuple[tf.Tensor]],
-        data_signature: Dict[Text, List[FeatureSignature]],
+        batch: List[tf.Tensor], data_signature: Dict[Text, List[FeatureSignature]]
```

Comment:
it can be `np.ndarray` (e.g. when we calculate `tf_label_data`) also all the batch functions will work with `np.ndarray` as well, let's keep it in type